src/java.base/share/classes/java/security/Key.java

```diff
@@ -121,9 +121,8 @@ public interface Key extends java.io.Serializable {
     /**
      * Returns the standard algorithm name for this key. For
      * example, "DSA" would indicate that this key is a DSA key.
```

Comment:
Suggest changing this to something more commonly in use than "DSA" like "RSA" or "EC".